src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-tabs/user-tabs.component.html

```diff
@@ -1,12 +1,14 @@
-<tabset>
-  <tab heading="Users"
-       i18n-heading
-       [active]="url === '/user-management/users'"
-       (selectTab)="navigateTo('/user-management/users')">
-  </tab>
-  <tab heading="Roles"
-       i18n-heading
-       [active]="url === '/user-management/roles'"
-       (selectTab)="navigateTo('/user-management/roles')">
-  </tab>
-</tabset>
+<ul ngbNav
+    #nav="ngbNav"
+    [activeId]="router.url"
+    (navChange)="router.navigate([$event.nextId])"
+    class="nav-tabs">
```

Comment:
Just wondering if it would make sense to have a `cd-tabs` directive which can simply be configured by a json like with `cd-datatable`, this would reduce the now a lot longer HTML files and it's harder to mess the HTML up by forgetting the right class at the right place.

Reply:
I would prefer to do this after we merge this, #34889 and #34946.
Then we can make sure we can still do all the current features with 1 component.